### PR TITLE
2.2.0 dev

### PR DESCRIPTION
**Pull request by kcw-grunt**
Please review
@losh11 @thrasher-

### CODE REVIEW

**Comment by kcw-grunt** [comment]
sorry...I wanted to double check

**Comment by kcw-grunt** [comment]
Tried to squash but the merge conflicts looks like more than a notion.

File: loafwallet/BRAPIClient+Wallet.swift [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 import Foundation
 
-private let ratesURL = "https://litecoin.com/api/v1/rates"
-private let fallbackRatesURL = "https://api.loshan.co.uk/api/v1/rates"
+private let ratesURL = "https://api.loshan.co.uk/api/v1/rates"
```
Comment (thrasher-) [inline loafwallet/BRAPIClient+Wallet.swift]: Want to double check that this is intentional and not using loafwallet as the primary rates URL?

File: loafwallet/BRAPIClient+Wallet.swift [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
+private let fallbackRatesURL = "https://api.loafwallet.org/api/v1/rates"
```
Comment (thrasher-) [inline loafwallet/BRAPIClient+Wallet.swift]: Just checked this URL, I can see why the above is done. Getting a GET request error on this endpoint.

**Comment by kcw-grunt** [comment]
I’ll check when I get to my Mac.


Kerry
-Sent via iPhone, pleaze excuse mah typoes-

**Pull request merged**